src/components/navbar.tsx

```diff
@@ -64,7 +64,9 @@ const Navbar = () => {
             >
               <ColorModeToggle />
             </HStack>
-            <WalletConnectModal />
+
+            <WalletConnect />
+
```

Comment:
swapped the modal component out for the WalletConnect dispatch component.. (maybe WalletConnectDispatch is a better name :thinking: ??!)